src/components/schedule-page/validation-drawer/error-list.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ScheduleErrorMessageModel } from "../../../common-models/schedule-error-message.model";
+import ErrorListItem from "./error-list-item.component";
+
+interface Options {
+  errors?: ScheduleErrorMessageModel[];
+}
+
+export default function ErrorList({ errors = [] }: Options): JSX.Element {
+  return (
+    <div>
+      {errors?.length > 0 &&
+        errors.map(
+          (error): JSX.Element => (
+            <ErrorListItem key={Math.random().toString().substr(2, 9)} error={error} />
```

Comment:
Ja bym lepiej użył error.substr(2,9) + index.
Bo nikt nie daje gwarancji, że losowe liczby się nie powtórzą ;) 
Natomiast w tym przypadku o którym pisałem, mamy gwarancję, że klucze będą unikatowe dla każdej pozycji listy.

Reply:
zmieniłam, ale nie jestem pewna czy to jest dobrze, bo:
![errorToString](https://user-images.githubusercontent.com/48785655/102011174-eee43080-3d42-11eb-9ea8-3960dd2cf2d4.png)
a z artykułu, który mi wysłałeś:
![index as a key](https://user-images.githubusercontent.com/48785655/102011198-14713a00-3d43-11eb-91ca-f873fd923deb.png)